### PR TITLE
update code of conduct to version 1.4

### PR DESCRIPTION
**Pull request by weissi**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,55 +1,36 @@
...
-As contributors and maintainers of this project, and in the interest of fostering an open and
-welcoming community, we pledge to respect all people who contribute through reporting
-issues, posting feature requests, updating documentation, submitting pull requests or patches,
-and other activities.
-
-We are committed to making participation in this project a harassment-free experience for
-everyone, regardless of level of experience, gender, gender identity and expression, sexual
-orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or
-nationality.
+To be a truly great community, the SwiftNIO project needs to welcome developers from all walks of life, with different backgrounds, and with a wide range of experience. A diverse and friendly community will have more great ideas, more unique perspectives, and produce more great code.  We will work diligently to make the SwiftNIO community welcoming to everyone.
```
Comment (glbrntt) [inline CODE_OF_CONDUCT.md:1]: We're missing `# Code of Conduct` now -- is that intentional?

Comment (weissi) [reply CODE_OF_CONDUCT.md:1]: @glbrntt no! thank you, updating all.

Comment (weissi) [reply CODE_OF_CONDUCT.md:1]: great catch, should be fixed. thanks

**Review by glbrntt — APPROVED** [review]

**Comment by ModestGoblin** [comment]
> 

image did they block me 🥺

**Review by Lukasa — APPROVED** [review]

**Pull request merged**